clr-k8s-examples/create_stack.sh

```diff
@@ -144,10 +146,10 @@ function cni() {
 		FLANNEL_VER=${1:-$FLANNEL_VER}
 		FLANNEL_URL="https://github.com/coreos/flannel"
 		FLANNEL_DIR="0-flannel"
-		
+
```

Comment:
suggestion: maybe put whitespace fixes in a separate commit in the future - it makes reviewing the difference between a whitespace fix and a technical fix easier ;-)

Reply:
I usually after doing changes do a shfmt check.